tools/functional-tester/etcd-tester/main.go

```diff
@@ -49,6 +51,8 @@ func main() {
 	consistencyCheck := flag.Bool("consistency-check", true, "true to check consistency (revision, hash)")
 	isV2Only := flag.Bool("v2-only", false, "'true' to run V2 only tester.")
 	stresserType := flag.String("stresser", "default", "specify stresser (\"default\" or \"nop\").")
+	failureTypes := flag.String("failures", "default,failpoints", "specify failures (concat of \"default\" and \"failpoints\").")
+	terminate := flag.Bool("terminate", true, "'true' to terminate an etcd cluster after testing.")
```

Comment:
Yeah, the failure decoupling looks OK overall. The namazu/script failure configuration (if it makes sense to do it this way) can probably be done via something like `--failures=default,wrapper=path-to-script` in exchange for slightly complicating the `--failures` parsing or with an extra option `--failure-wrapper=path-to-script` if the extra parsing smarts turns out to be obnoxious.